src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -141,7 +133,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		)
 
 		return RiskProvider(
-			configuration: config,
+			configuration: RiskProvidingConfiguration.defaultConfiguration,
```

Comment:
_Swiftier_ naming:
```suggestion
			configuration: .default,
```

If accepted: needs also adjustments to code below!

Reply:
done